modules/swagger-codegen/src/main/resources/rust/api.mustache

```diff
@@ -0,0 +1,103 @@
+{{>partial_header}}
+use std::rc::Rc;
+use std::borrow::Borrow;
+
+use hyper;
+use serde_json;
+use futures;
+use futures::{Future, Stream};
+
+use super::{Error, configuration, models};
+
+pub struct {{{classname}}}Impl<C: hyper::client::Connect> {
+    configuration: Rc<configuration::Configuration<C>>,
+}
+
+impl<C: hyper::client::Connect> {{{classname}}}Impl<C> {
+    pub fn new(configuration: Rc<configuration::Configuration<C>>) -> {{{classname}}}Impl<C> {
+        {{{classname}}}Impl {
+            configuration: configuration,
+        }
+    }
+}
+
+pub trait {{classname}} {
+{{#operations}}
+{{#operation}}
+    fn {{{operationId}}}(&self, {{#allParams}}{{paramName}}: {{{dataType}}}{{#hasMore}}, {{/hasMore}}{{/allParams}}) -> Box<Future<Item = {{^returnType}}(){{/returnType}}{{#returnType}}{{{returnType}}}{{/returnType}}, Error = Error>>;
+{{/operation}}
+{{/operations}}
+}
+
+
+impl<C: hyper::client::Connect>{{classname}} for {{classname}}Impl<C> {
+{{#operations}}
+{{#operation}}
+    fn {{{operationId}}}(&self, {{#allParams}}{{paramName}}: {{#isString}}&str{{/isString}}{{^isString}}{{{dataType}}}{{/isString}}{{#hasMore}}, {{/hasMore}}{{/allParams}}) -> Box<Future<Item = {{^vendorExtensions.x-returnTypeInMethod}}(){{/vendorExtensions.x-returnTypeInMethod}}{{#vendorExtensions.x-returnTypeInMethod}}{{{.}}}{{/vendorExtensions.x-returnTypeInMethod}}, Error = Error>> {
+        let configuration: &configuration::Configuration<C> = self.configuration.borrow();
+        let mut req = hyper::Request::new(
+            hyper::Method::{{httpMethod}},
+            format!("{}{{path}}", configuration.base_path{{#allParams}}, {{paramName}}={{paramName}}{{/allParams}}));
```

Comment:
So yes, answering your question, only path parameters and query parameters (if any) will end up in here.